### PR TITLE
ui: show tooltips for buttons with no text

### PR DESCRIPTION
**Pull request by zx2c4**
This allows tooltips to explain what a button does in a mixed text and image bar.

### CODE REVIEW

**Comment by jscholes** [comment]
If a button has text defined but displays an image, it would be good to ensure that the text is spoken by screen readers.  A tool tip probably won't accomplish that, and it may be necessary to explicitly update the accessibility properties of the control.  I'd need to see a working sample first, to see if the text was already spoken (or in other words if Windows was smart enough to associate any text with the button independently of the tool tip usage).  But I'm just flagging it up on your radar.

**Comment by zx2c4** [comment]
You're mistaken. This has nothing to do with "buttons that have Text() defined". This is only valid for buttons that do not have Text() defined. The win32 toolbar control does not show tooltips for buttons that have Text() defined.

Actually, the way this works is:

Text() defined --> PszText: Text(), FsStyle: TB_SHOWTEXT
Text() not defined, ToolTip() defined --> PszText: ToolTip(), FsStyle: 0

In other words, if FsStyle does not have TB_SHOWTEXT, then win32 uses PszText as the tooltip rather than as the displayed button text. I assume, therefore, that win32 will do the right thing accessibility-wise.
